.circleci/config.yml

```diff
@@ -1,53 +1,12 @@
 version: 2.1
 
 orbs:
-  plugin-ci: mattermost/plugin-ci@volatile
+  plugin-ci: mattermost/plugin-ci@0.1.0
 
 workflows:
   version: 2
-  nightly:
-    triggers:
-      - schedule:
-          cron: "0 0 * * *"
-          filters:
-            branches:
-              only:
-                - master
```

Comment:
Why did this file go changed? It should use the version from https://github.com/mattermost/mattermost-plugin-demo/blob/master/.circleci/config.yml

Reply:
Did you mean to link to the starter-template instead?

Reply:
nvm I see your comment [here](https://github.com/mattermost/mattermost-plugin-todo/pull/103#discussion_r462933898) explaining why :+1:

Reply:
This is a point of confusion that we should address.  Thanks for the linking the comment regarding why we sync this file with demo instead of the starter template